requirements/pip.txt

```diff
@@ -47,6 +51,9 @@ dnspython==1.15.0
 
 # VCS
 httplib2==0.11.3
+
+# GitPython 2.1.11 makes TestGitBackend.test_git_tags to fail because
+# of an UnicodeError
```

Comment:
Maybe mention that we can update when we drop py2 support

Reply:
I'm starting to feel that this package is too unstable to rely all our building process on it. We just updated a bugfix version and it breaks our flow.

In the requirements, it says that it supports 2.7 or newer: https://gitpython.readthedocs.io/en/stable/intro.html#requirements

This is the commit that we think that introduced the issue: https://github.com/gitpython-developers/GitPython/commit/7f08b7730438bde34ae55bc3793fa524047bb804

Reply:
More examples:

* https://github.com/gitpython-developers/GitPython/issues/761
* https://github.com/gitpython-developers/GitPython/issues/279
* https://github.com/gitpython-developers/GitPython/issues/774 (the last comment of this one tells us that Unicode may never be well supported :( )

I suppose that we should start thinking about another solution, unfortunately.

Reply:
Tests in `master` and `2.1.11` fail. Also in 2.1.10... :disappointed_relieved:

Reply:
Regarding dropping py27 support, this will be our v3.0 -- it might make sense to time 3.0 for all of our deprecations in jan 2019. i opened #4594 to discuss more